CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Cromwell Change Log
 
+## 51 Release Notes
+
+### New functionality
+
+#### new xxh64 and fingerprint strategies for call caching
+
+To alleviate problems with existing call cache strategies, two new strategies for files on a shared filesystem (ie not in cloud storage) have been added: `xxh64` and 
```

Comment:
Can you be specific on what problems are being alleviated and how? Would the following be correct?
```suggestion
Existing call cache strategies can be slow on shared filesystems (SFS backend, i.e. not in cloud storage). Two faster strategies have been added for this use case: `xxh64` and 
```

Reply:
Well `path` and `path+modtime` are already extremely fast, but they don't work with docker. Md5 is the slow strategy. Thanks for the suggestion. I have updated the description.